frontend/src/types/media.ts

```diff
@@ -44,6 +44,8 @@ export interface Media {
 
   tags: Tag[]
   fields_matched?: string[]
+
+  mature: boolean
```

Comment:
Wow, I didn't know we didn't have this in the type! Great catch! Is it always available, or should we mark it as optional?

Reply:
It's always available.